habittracker/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from datetime import datetime
+from django.db.models import UniqueConstraint
+
+class User(AbstractUser):
+    def __repr__(self):
+        return f"<User username={self.username}>"
+
+    def __str__(self):
+        return self.username
+
+class Habit(models.Model):
+    name = models.CharField(null = True, max_length=200)
+    overall_goal = models.IntegerField(default = 0)
+    description = models.TextField(max_length=1000)
+    app_user = models.ForeignKey(User, on_delete=models.CASCADE, null=True, blank=True, related_name="goal_setter")
+    
+
+    def __str__(self):
+        return str(self.overall_goal)
```

Comment:
This is going to make it hard to read -- just returning an integer. How about something like 
```
def __str__(self):
        return f"{self.name} goal={self.overall_goal}"
   ```